### PR TITLE
Update grafana dev dependencies to v8.5.10

### PR DESCRIPTION
**Pull request by kevinwcyu**
Separating out the dev dependency updates from https://github.com/grafana/redshift-datasource/pull/177 into a separate PR.

### CODE REVIEW

File: src/ConfigEditor/ConfigEditor.tsx [hunk 1]
```diff
@@ -84,6 +84,7 @@ export function ConfigEditor(props: Props) {
         });
       });
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (sarahzinger) [inline src/ConfigEditor/ConfigEditor.tsx:87]: is this needed? seems like nothing is changing? maybe just fixing an old lint error?

File: src/ConfigEditor/ConfigEditor.tsx [hunk 2]
```diff
@@ -227,7 +229,6 @@ export function ConfigEditor(props: Props) {
               },
             })
           }
-          css={undefined}
```
Comment (sarahzinger) [inline src/ConfigEditor/ConfigEditor.tsx:230]: 😆

File: src/__mocks__/datasource.ts [hunk 3]
```diff
@@ -8,6 +8,7 @@ export const mockDatasource = new DataSource({
   type: 'redshift-datasource',
   name: 'Redshift Data Source',
   jsonData: {},
+  access: 'direct',
```
Comment (sarahzinger) [inline src/__mocks__/datasource.ts:11]: what does this do again?

**Review by sarahzinger — APPROVED** [review]
code seems good! I didn't do any manual testing but so long as it builds ok for you let's ship! 🚀

File: src/__mocks__/datasource.ts (hunk 3 above)
Comment (kevinwcyu) [reply src/__mocks__/datasource.ts:11]: not really sure, think the type just expects an access property.

File: src/ConfigEditor/ConfigEditor.tsx (hunk 1 above)
Comment (kevinwcyu) [reply src/ConfigEditor/ConfigEditor.tsx:87]: probably updated packages pulls in new eslint rules?

**Pull request merged**